content/en/software/install-and-run/linux/linux-local-from-github.md

```diff
@@ -0,0 +1,19 @@
+﻿﻿---
+title: Download WSL (Local Installation) Install Files from Github
+weight: 40
+toc_hide: true
+hide_summary: true
+type: docs
+---
+<style>.td-content blockquote { border-left: none; color: inherit; padding-left: 2rem;}</style>
+
+1.  Go to the [Releases page for Interlisp/Medley on github.com](https://github.com/interlisp/medley/releases).  Here you will see a list of releases in reverse chronological order.  
+ <div align="center"><img alt="Github Releases Page" width=500 src="../images/releases-page-landing.png"><p>&nbsp;</p></div>
+
+3. Choose the release you are interested in (most likely the release marked ***Latest***) and click on the arrow next the word ***Assets*** to reveal the list of assets (i.e., installation files) for that release.   
```

Comment:
Change this to 2. The `<div />` block above this breaks the list and the markdown processors starts up at three.  You can either change the 4 to 3 or leave it.  It will be processed correctly.